### DISCUSSION TITLE
Possible confusions between local pp & global pp for the same score

### DISCUSSION THREAD

**Discussion by Mrcubix** [ppy/osu#30370] [Q&A] [answered]
The same score appear to have different values depending on whether you check if from the local or global tab of a beatmap.

Local: 

![image](https://github.com/user-attachments/assets/db38a1ad-5900-426a-95a4-0aad8dee5b6d)

Global:

![image](https://github.com/user-attachments/assets/12d81962-2fda-4a0b-96f1-bf78b2dd4abb)

I feel like it should be the same value in both cases here.

**Comment by bdach** [comment, marked as answer]
The value will be the same once the pp changes that have been advertised *everywhere* roll out to completion.

**Reply by Mrcubix** [reply]
The real question is, by same, same as which. which one is correct here, for now.

**Reply by bdach** [reply]
Well eventually both places should be using the 'local' number as you put it.
